doc/dependencies.md

```diff
@@ -22,6 +22,7 @@ These are the dependencies currently used by Bitcoin Core. You can find instruct
 | Python (tests) |  | [3.4](https://www.python.org/downloads) |  |  |  |
 | qrencode | [3.4.4](https://fukuchi.org/works/qrencode) |  | No |  |  |
 | Qt | [5.9.6](https://download.qt.io/official_releases/qt/) | 5.x | No |  |  |
+| systemtap | [4.7](https://sourceware.org/systemtap/) |  |  |  |  |
```

Comment:
Bitcoin does not seem to define a specific minimum version.

```suggestion
| [systemtap](https://sourceware.org/systemtap/) ([tracing](tracing.md))| N/A |
```

Reply:
This is our "version used" column. Bitcoin's table has been simplified a lot. Ours still has "version used" and "minimum version" separately.